[0m10:25:04.244442 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1075166e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1083f6260>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1083f5ba0>]}


============================== 10:25:04.259463 | d544428b-b530-4a28-98a9-8fe08cf9f329 ==============================
[0m10:25:04.259463 [info ] [MainThread]: Running with dbt=1.9.3
[0m10:25:04.259771 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/ricky/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt init dbt_cosmos', 'send_anonymous_usage_stats': 'True'}
[0m10:25:04.283540 [debug] [MainThread]: Starter project path: /Users/ricky/astro-cosmos/.venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m10:25:04.286505 [info ] [MainThread]: 
Your new dbt project "dbt_cosmos" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:25:04.286694 [info ] [MainThread]: Setting up your profile.
[0m10:25:21.122346 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 16.912098, "process_in_blocks": "0", "process_kernel_time": 0.07254, "process_mem_max_rss": "117243904", "process_out_blocks": "0", "process_user_time": 0.618559}
[0m10:25:21.122696 [debug] [MainThread]: Command `dbt init` succeeded at 10:25:21.122630 after 16.91 seconds
[0m10:25:21.122917 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1075166e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107ec10c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1083abf40>]}
[0m10:25:21.123202 [debug] [MainThread]: Flushing usage events
[0m10:25:21.842803 [debug] [MainThread]: An error was encountered while trying to flush usage events
